### ISSUE TITLE
Goodcheck fails on empty files

### ISSUE THREAD

**Issue by dcwither** [sider/goodcheck#72]
Steps to Repro:
1. create empty file `x.js`
2. add goodcheck to Gemfile
3. `bundle install`
4. `bundle exec goodcheck check --debug x.js`

goodcheck.yml
```
rules:
  # id, pattern, message are required attributes.
  - id: example.github
    pattern: Github
    message: Do you want to write GitHub?
    glob:
      - '**/*.rb'
      - '**/*.{yaml,yml}'
      - 'public/**/*.html'
    fail:
      - Signup via Github
    pass:
      - Signup via GitHub

  # You can have *justification* to explain the exceptional cases.
  - id: example.localStorage
    pattern:
      token: localStorage
    message: |
      Using localStorage may raise error (example: with Safari in private mode)
    justification:
      - If you catch the errors.
      - When you implement admin console, where end users won't access.
    glob:
      - '**/*.js'
    fail:
      - |
        localStorage.setItem(key, value);

  # You can put `not` pattern to detect something is missing.
  - id: example.strict-mode
    not:
      pattern: use strict
    message: Use *strict mode* if possible.
    glob: '**/*.js'
    fail:
      - |
        const var = "This is *sloppy* mode."

  # You can omit pattern, which prints the message on the files specified by glob.
  - id: example.package-lock.json
    message: Some of the packages are updated!
    justification:
      - If you update some of the packages.
    glob: 'package-lock.json'
# You can import rules.
# import:
#   - https://example.com/example-rules.yml

# You can skip checking files.
# exclude:
#   - node_modules
#   - vendor
```
Expected behavior:
Fails `strict-mode` check

Actual Behavior:
```
> bundle exec goodcheck check --debug x.js
[2.4.1] Configuration = goodcheck.yml
[2.4.1] Rules = []
[2.4.1] Format = 
[2.4.1] Targets = [x.js]
[2.4.1] Force download = false
[2.4.1] Home path = /home/dev/.goodcheck
[2.4.1] Loading configuration: goodcheck.yml
[2.4.1] [goodcheck.yml] Loading rule: example.github
[2.4.1] [goodcheck.yml] Loading rule: example.localStorage
[2.4.1] [goodcheck.yml] Loading rule: example.strict-mode
[2.4.1] [goodcheck.yml] Loading rule: example.package-lock.json
[2.4.1] Checking target: x.js
[2.4.1] [x.js] Checking file: x.js
[2.4.1] [x.js] [x.js] Checking rule: example.localStorage
[2.4.1] [x.js] [x.js] Checking rule: example.strict-mode
#<NoMethodError: undefined method `chomp' for nil:NilClass>
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/reporters/text.rb:33:in `issue'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:39:in `block (5 levels) in run'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/analyzer.rb:58:in `scan_simple'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/analyzer.rb:30:in `scan'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:36:in `block (4 levels) in run'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/reporters/text.rb:19:in `rule'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:34:in `block (3 levels) in run'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:73:in `block (6 levels) in each_check'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/config.rb:38:in `block (2 levels) in rules_for_path'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/config.rb:30:in `each'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/config.rb:30:in `block in rules_for_path'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/config.rb:14:in `each'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/config.rb:14:in `each_rule'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/config.rb:29:in `each'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/config.rb:29:in `map'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/config.rb:29:in `rules_for_path'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:60:in `block (5 levels) in each_check'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/reporters/text.rb:15:in `file'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:57:in `block (4 levels) in each_check'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/activesupport-5.2.2.1/lib/active_support/tagged_logging.rb:71:in `block in tagged'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/activesupport-5.2.2.1/lib/active_support/tagged_logging.rb:28:in `tagged'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/activesupport-5.2.2.1/lib/active_support/tagged_logging.rb:71:in `tagged'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:56:in `block (3 levels) in each_check'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:108:in `each_file'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:54:in `block (2 levels) in each_check'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/activesupport-5.2.2.1/lib/active_support/tagged_logging.rb:71:in `block in tagged'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/activesupport-5.2.2.1/lib/active_support/tagged_logging.rb:28:in `tagged'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/activesupport-5.2.2.1/lib/active_support/tagged_logging.rb:71:in `tagged'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:53:in `block in each_check'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:51:in `each'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:51:in `each_check'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:31:in `block (2 levels) in run'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/reporters/text.rb:11:in `analysis'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:29:in `block in run'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/config_loading.rb:15:in `handle_config_errors'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/commands/check.rb:26:in `run'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/cli.rb:103:in `check'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/lib/goodcheck/cli.rb:29:in `run'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/goodcheck-2.4.1/exe/goodcheck:7:in `<top (required)>'
  /home/dev/.rbenv/versions/2.5.5/bin/goodcheck:23:in `load'
  /home/dev/.rbenv/versions/2.5.5/bin/goodcheck:23:in `<top (required)>'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/cli/exec.rb:74:in `load'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/cli/exec.rb:74:in `kernel_load'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/cli/exec.rb:28:in `run'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/cli.rb:463:in `exec'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/cli.rb:27:in `dispatch'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/cli.rb:18:in `start'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/bundler-2.0.1/exe/bundle:30:in `block in <top (required)>'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/friendly_errors.rb:124:in `with_friendly_errors'
  /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/bundler-2.0.1/exe/bundle:22:in `<top (required)>'
  /home/dev/.rbenv/versions/2.5.5/bin/bundle:23:in `load'
  /home/dev/.rbenv/versions/2.5.5/bin/bundle:23:in `<main>'
```

This is not a critical issue since there's not much point to empty JS files, but I noticed my project has them, and found this issue. I figure it qualifies as a minor stability issue.

**Comment by ybiquitous** [comment]
Thanks for your report! I could reproduce it. This a bug. We will fix it soon.

**Comment by ybiquitous** [comment]
@dcwither I fixed the issue. If you're using Docker, you can try via the following command soon:

```sh
$ cd your_working_directory
$ docker run -t --rm -v "$(pwd):/work" sider/goodcheck:master check
```

See https://github.com/sider/goodcheck#docker-images

**Comment by ybiquitous** [comment]
Released [v2.4.2](https://github.com/sider/goodcheck/releases/tag/v2.4.2) 🎉 

Thank you a lot! @dcwither
